Modelica/Electrical/Spice3.mo

```diff
@@ -4435,6 +4435,12 @@ on the model behaviour.
 </html>"));
   end Types;
 
+  package Constants "Spice3-specific constants"
+    extends Modelica.Icons.Package;
+
+    constant Modelica.Units.SI.Time infiniteTime= 1e20 "Numerically safe very large time horizon (3e12 years)";
```

Comment:
Instead of the vague "numerically safe", could you point to the specific problem this solves? E.g. something like "Very large time horizon (3e12 years) that avoids overflowing values in time computations"?

Reply:
Sure

Reply:
Done.